src/pypit.py

```diff
@@ -12,7 +12,8 @@
 # Import PYPIT routines
 import ardebug
 debug = ardebug.init()
-#debug['develop'] = True
+debug['develop'] = True
```

Comment:
This should be commented out when pushing to GitHub (not all users will want the develop setting on).